src/components/MovieDetailsPage/MovieDetailsPage.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import {useParams} from "react-router-dom";
+import axiosInstance from '../../api/api';
+import MovieDetails from "../MovieDetails/MovieDetails";
+import './MovieDetailsPage.scss';
+
+const MovieDetailsPage = () => {
+    let { movieId } = useParams();
+    const [selectedMovie, setSelectedMovie] = useState(null);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const response = await axiosInstance.get(
```

Comment:
Do you want to use loader Instead of fetching movie in your component? It will be much cleaner.

https://reactrouter.com/en/main/start/tutorial#loading-data